js/index.js

```diff
@@ -59,6 +90,144 @@ function showSkills() {
   }
 }
 
+function handleProjectsData(e) {
+  let projectsRequest = e.target;
+  if (projectsRequest.readyState === 4 && projectsRequest.status === 200){
+    renderProjectsData(projectsRequest);
+  }
+  else{
+    console.error("Failed to load JSON data from the given URL");
+  }
+}
+
+function renderProjectsData(projectsRequest){
+  //VARIABLES
+  //Request variable
+  let projectsRequestJSON = JSON.parse(projectsRequest.responseText);
+  //Project's aux variables
+  let projectImages=["burger-city-restaurant.png", "dental-clinic.png", "js-animation.png", "personal-portfolio.png"];
```

Comment:
Instead of having project specific metadata relying on the projects being listed in a certain order, I'd 
create a mapping of project id or project name to it's supporting data: 
```javascript
PROJECT_DATA_MAP = {
       "Burger-City-Restaurant": {
            image: "burger-city-restaurant.png",
            dates: "July 2020 - December 2020",
            tools: ["HTML5", "CSS3", "JS", "Bootstrap", "ASP.NET"] 
        }
    }
```

Reply:
That's a great idea! I will make sure to add it on the final project submission. Thank you!